src/bsoncxx/stdx/optional.hpp

```diff
@@ -40,6 +40,7 @@ BSONCXX_INLINE_NAMESPACE_END
 
 #include <boost/none.hpp>
 #include <boost/optional/optional.hpp>
+#include <boost/optional/optional_io.hpp>
```

Comment:
To check, this include was needed to use `bsoncxx::stdx::optional` in test assertions like `CHECK(service_id)`, right?

Reply:
Correct, otherwise Catch2 doesn't know how to display them.